implementation/src/main/java/io/smallrye/jwt/auth/principal/DefaultJWTParser.java

```diff
@@ -130,4 +142,9 @@ private void setKeyEncryptionAlgorithmIfNeeded(JWTAuthContextInfo newAuthContext
             newAuthContextInfo.setKeyEncryptionAlgorithm(newAlgo);
         }
     }
+
+    private SecretKey createSecretKeyFromSecret(String secret) {
+        byte[] secretBytes = secret.getBytes(StandardCharsets.UTF_8);
+        return new SecretKeySpec(secretBytes, "AES");
+    }
```

Comment:
Duplicate of `JwtBuildUtils.createSecretKeyFromSecret` (FYI)

Reply:
Hi Mike @MikeEdgar thanks for spotting it, yes, I recall thinking about, I wanted to keep the `build` related code  as isolated as possible in case we consider in time moving it in its own module, but now I've realized this code really belongs in `KeyUtils` :-). So I've just moved it there, and will merge shortly, cheers